### DISCUSSION TITLE
Certain drawings in a symbol seem to glitch and not allow editing

### DISCUSSION THREAD

**Discussion by Pitchblack70** [keiwando/toonsquid#402] [Q&A]
I've had this happen more than once.

And when it does, not only can't one draw or erased on THAT particular drawing...

But it seems to "disappear" in the project.

Is this a known glitch and fixable?  Thanks!!  

I'm very much appreciating and enjoying Toonsquid, except for frustrating oddities like this!  :)

**Comment by keiwando** [comment]
I haven't heard of this before, but if you can reproduce it, let me know of the steps or if you happen to run into it - if possible - export the project file and send it to contact@toonsquid.com so I can take a closer look at what might be going on. Alternatively create a screen recording that shows as many details as possible, e.g. the layer types and layer structure etc.
Thanks!

**Reply by Pitchblack70** [reply]
Thanks - will do!  I'll be diving into a new project soon, so I'll keep an eye out for the glitch happening, and document all those factors (vs. using older projects where I can't recall everything that was done at this point!)
